d3a_api_client/utils.py

```diff
@@ -220,3 +223,17 @@ def execute_function_util(function: callable, function_name):
     except Exception as e:
         logger.error(
             f"{function_name} raised exception: {str(e)}. \n Traceback: {str(traceback.format_exc())}")
+
+
+def iterate_over_all_client_setup():
+    module_list = []
+    client_modules_path = setups.__path__
+    for loader, module_name, is_pkg in walk_packages(client_modules_path):
+        if is_pkg:
+            loader.find_module(module_name).load_module(module_name)
+        else:
+            module_list.append(module_name)
+    return module_list
```

Comment:
How about instead of copying this function, to move it from d3a repository to d3a-interface ? That way you will be able to reuse it in both d3a and d3a-api-client repositories. I think you also need to add one extra function argument for the modules path, but it is fairly straightforward to do so